[FW][FIX] _payment_group: add_all not editing aml on m2m

fw-bot-adhoc opened the pull request:
Por lo que parece un bug del framework, cuando el método computado _compute_to_pay_move_lines agrega las lineas Luego la interfaz trata de hacer un write en los amls lo que termina siendo malo a nivel performance pero además peligroso porque termina corriendo update de cosas que no queremos (y eventualmente dando error).

Si se usan los botones add_all esto no pasa, es un tema del compute. En v13 lo teníamos como onchange, tal vez por eso no pasaba.

Resolvemos forzando que en la vista todo sea redonly

Forward-Port-Of: ingadhoc/account-payment#431

jjscarafia committed:
[FIX] _payment_group: add_all not editing aml on m2m

Por lo que parece un bug del framework, cuando el método computado _compute_to_pay_move_lines agrega las lineas
Luego la interfaz trata de hacer un write en los amls lo que termina siendo malo a nivel performance pero además peligroso porque termina corriendo update de cosas que no queremos (y eventualmente dando error).

Si se usan los botones add_all esto no pasa, es un tema del compute. En v13 lo teníamos como onchange, tal vez por eso no pasaba.

Resolvemos forzando que en la vista todo sea redonly

X-original-commit: 2b823ab25fb8bf8e764dca9d3c9c7deec1cef320

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/432).

fw-bot-adhoc commented:
@jjscarafia cherrypicking of pull request ingadhoc/account-payment#431 failed.

stdout:
```
CONFLICT (modify/delete): account_payment_group/views/account_move_line_view.xml deleted in HEAD and modified in 877c74c7 ([FIX] _payment_group: add_all not editing aml on m2m). Version 877c74c7 ([FIX] _payment_group: add_all not editing aml on m2m) of account_payment_group/views/account_move_line_view.xml left in tree.
CONFLICT (modify/delete): account_payment_group/__manifest__.py deleted in HEAD and modified in 877c74c7 ([FIX] _payment_group: add_all not editing aml on m2m). Version 877c74c7 ([FIX] _payment_group: add_all not editing aml on m2m) of account_payment_group/__manifest__.py left in tree.

```

stderr:
```
16:23:57.138641 git.c:444               trace: built-in: git cherry-pick 877c74c7ae0e3c14cfca312a052874ebb904ba52
error: could not apply 877c74c7... [FIX] _payment_group: add_all not editing aml on m2m
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port

jjscarafia commented:
esto vamos a ver de llevarlo a mano en la tarea
